build.assets/Dockerfile-babybox

```diff
@@ -0,0 +1,44 @@
+FROM golang:1.20
+
+# Image layers go from less likely to most likely to change.
+RUN apt-get update && \
+  apt-get install -y --no-install-recommends \
+    npm \
+    unzip \
+    && \
+  rm -rf /var/lib/apt/lists/*
+
+# protoc-gen-gogofast
+# Keep in sync with api/proto/buf.yaml (and buf.lock)
+ARG GOGO_PROTO_TAG=v1.3.2
```

Comment:
Shouldn't the values be used from the makefile or fail if they are not set? Otherwise, we may keep them out of sync and don't even know.

Reply:
This is handy for direct "docker build"s, but I can remove the defaults. Done.

Reply:
Thanks